discord_bot/chat_handler.py

```
@@ -1,100 +1,302 @@
                 logger.info("[on_ready] Message sync completed")
-                
+
             except Exception as e:
                 logger.error(f"[on_ready] Backfill/sync task failed: {e}", exc_info=True)
-        
-        logger.info(f"[on_ready] Creating backfill+sync background task for {len(text_channels)} channels...")
+
+        logger.info(
+            f"[on_ready] Creating backfill+sync background task for {len(text_channels)} channels..."
+        )
         asyncio.create_task(run_backfill_and_sync())
```

Ruff (0.15.2)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)